examples/lessons/13/app.js

```diff
@@ -274,54 +287,6 @@ function animate(state) {
   state.lastTime = timeNow;
 }
 
-/* global document */
-// eslint-disable-next-line complexity
-function getControlValues() {
```

Comment:
Instead of deleting these, should we restore the HTML elements above instead, they seem to have been cut by mistake in a previous cleanup.

Reply:
It actually looks like that logic has been moved around for a while, but hasn't been hooked into any actual UI for a long while:
https://github.com/uber/luma.gl/blob/5.0-release/examples/lessons/13/app.js
https://github.com/uber/luma.gl/blob/6.0-release/examples/lessons/13/app.js

I'll leave this for now, and we can discuss we want to flesh out with controls.